ts/src/krakenfutures.ts

```diff
@@ -2142,6 +2139,7 @@ export default class krakenfutures extends Exchange {
         }
         const url = this.urls['api'][api] + query;
```

Comment:
@pcriadoperez I think something went wrong, this is the `krakenfutures` file 😅

Reply:
It looks like in another branch (https://github.com/ccxt/ccxt/pull/19836/files#diff-0f6be5c50c2d3c3beb8b6a55b945b3325e30ac7ed6085e3c7846b06fb28dc510)